layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts

```diff
@@ -0,0 +1,54 @@
+/*

```

Comment:
Any reason to copy them and not use them directly from the source code?

Reply:
Yes, building the overlays with latest master throws compile errors as the Makefile has been modified in the meanwhile. Quicker to add them like this for the next release

Reply:
I agree it's quicker but I think it's a hack and brings in duplication that we will have to take care when we update. And in the current implementation, if we update the package and forget to remove these files we will end up shadowing the ones provided by the package.

Reply:
I will create an issue and deal with it when I have some time

Reply:
So to rephrase, my proposal would be to go with this PR and create an issue to follow-up with fixing the compile error

Reply:
I did the Makefile fix and now this PR adds the latest revision of overlays.
Please review